examples/webengine/hello-sdl/manifest.js

```diff
@@ -2,17 +2,17 @@
 export default {
     "entrypoint": "./index.html",
     "appIcon": "./test_icon_1.png",
-    "appId": "hello-webengine2",
-    "appName": "Hello WebEngine2",
-    "category": "DEFAULT",
+    "appId": "hello-webengine3",
+    "appName": "Hello WebEngine3",
+    "category": "WEB_VIEW",
     "additionalCategories": [],
     "locales": {
         "de_DE": {
-            "appName": "Hallo JS",
+            "appName": "Hallo WebEngine3",
             "appIcon": "./test_icon_1.png"
         }
     },
     "appVersion": "1.0.0",
-    "minRpcVersion": "6.0",
-    "minProtocolVersion": "5.0"
-};
\ No newline at end of file
+    "minRpcVersion": "6.2.0",
```

Comment:
```suggestion
    "minRpcVersion": "7.0.0",
```

Reply:
@SilvioDC-Ford fixed in https://github.com/smartdevicelink/sdl_javascript_suite/pull/264/commits/8035899de3a2b59eae39bf9806b90bdabd46cdc7